src/Common/AzurePSCmdlet.cs

```diff
@@ -747,18 +750,14 @@ protected void LogQosEvent()
                 return;
             }
 
+            _qosEvent.ParameterSetName = this.ParameterSetName;
```

Comment:
This is already done by https://github.com/Azure/azure-powershell-common/blob/b6a77a31d2c6bcf2036fdb99e9bc5aef26c6fd72/src/Common/AzurePSCmdlet.cs#L648
Why do it again?

Reply:
AutoRest cmdlet has issue for the value of ParameterSetName. That is why we need to re-assign.